src/cloud-formation/deploy-indexer-fallback.yaml

```diff
@@ -1,29 +1,29 @@
 ---
 parameters:
   BrokerCpu: 1024
-  BrokerImageVersion: '4.0.2'
+  BrokerImageVersion: '5.0.0'
```

Comment:
- [ ] Confirm completion of v5.0.0 broker build: https://github.com/econia-labs/emojicoin-dot-fun/actions/runs/13148255189
- [x] Confirm completion of v5.0.0 processor build: https://github.com/econia-labs/aptos-indexer-processors/actions/runs/13148041463
- [ ] Mark this comment resolved